### ISSUE TITLE
Have an environment variable for the frontend elastic host

### ISSUE THREAD

**Issue by bfabio** [italia/developers.italia.it#625]
The elastic host configuration used by the frontend is now in `assets/scripts/esQuery.config.js`. It would be nice to have it as an environment variable settable in `.env`.

**Comment by sebbalex** [comment]
I agree, we should inject it at build time

**Comment by bfabio** [comment]
Depends on #646.
